src/main/java/easytests/api/v1/controllers/UsersController.java

```diff
@@ -51,15 +51,26 @@ private Boolean isAdmin() {
                 .map(model -> this.usersMapper.map(model, User.class))
                 .collect(Collectors.toList());
     }
+    
     /**
      * create
      */
     /**
      * update
      */
-    /**
-     * show(userId)
-     */
+    @GetMapping("/{userId}")
+    public User show(@PathVariable Integer userId) throws NotFoundException, ForbiddenException {
+        final UserModelInterface userModel = this.usersService.find(userId);
+
+        if (userModel == null) {
+            throw new NotFoundException();
+        }
+
+        if (!this.isAdmin()) {
```

Comment:
move admin check before user one. We should not tell to none admin users that some user id exists.